pkg/resource/deploy/source.go

```diff
@@ -82,15 +82,15 @@ type RegisterResourceEvent interface {
 	SourceEvent
 	// Goal returns the goal state for the resource object that was allocated by the program.
 	Goal() *resource.Goal
+	// SupportsPartialStables returns true if the requestor supports partially-stable properties during preview.
```

Comment:
comment is mostly fine.  however, my preference would be that there is at least one good place where we doc more in depth what this means.  i.e. what is a "partially stable property".  it would be good to have at least one example somewhere in the comments to make the use case clear. 

(note: this may exist and i may not have gotten to it yet).

Reply:
Nope, this is good feedback. As I noted above, part of why this is a draft is b/c the docs need work